scalene/scalene_profiler.py

```diff
@@ -1068,10 +1071,12 @@ def process_cpu_sample(
                 # Ignore sleeping threads.
```

Comment:
Will this work correctly for task frames showing the `tident` of the loop processing thread?